### DISCUSSION TITLE
🩻

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#554] [Galaxy Trivia] [answered]
**NEKYID**
**REHAT**
**SТUGL**
**RINAB**

*answer with emojis*

**Comment by peterbaccaro** [comment]
KIDNEY 🧒🦵
HEART ❤️
GLUTS 🍑
BRAIN 🧠

**Comment by thuongtruong1009** [comment, marked as answer]
NEKYID → 🐕🔑👦 (key, kid, dog)
REHAT → 🏡👒🛋️ (house, hat, sofa)
SТUGL → 🚪🌳🐊 (gust, tree, alligator)
RINAB → 🎨🏛️🗿 (brain, art, museum)
